### PR TITLE
14.0.0 Beta 1

### PR DESCRIPTION
**Pull request by MorrisJobke**
### Must haves

* [x] #10238 Add support for sending the password for a share by Nextcloud Talk
    * nextcloud/spreed#1049 Add support for sending the password for a share by Nextcloud Talk (not blocking the release, because talk app is independent)

### Nice to have

* [x] #10310 Accessibility fixes for header and global elements (JS unit tests fail)
* [x] #10298 Only create the schema when moving between databases
* [x] nextcloud/firstrunwizard#71 UI polishing
* [x] #10322 Login: Implements the "user disabled" message like the "wrong password" message
* [x] #10377 Cache isCached state for scss resources
* [ ] #10221 Prevent too long identifier names (PHP unit tests fail for Postgres)
* [ ] #10320 Cleanup of leftover-sorting-code from quickaccess-feature
* ~~#10255 Add support for room shares~~ (moved to 15)

### Optional

* [x] #8380 Re-enable apps that got automatically disabled while updating
* [x] #9222 Allow to search files by comments
* [ ] #10030 Do not allow RTLO char in filenames
* ~~#10219 Fail the translation test when there are multiple single placeholders~~
* ~~#9633 Check proper case for user UID~~
* ~~#10248 Introduces avatars for guest users~~

### Known bugs (and potential fixes)

* [x] #10386 Favorites and Shares icon in sidebar misaligned
* [x] #10326 Fix regex of iconsCacher
* [x] #10372 Fix files header sticky positioning
* [x] https://github.com/nextcloud/activity/pull/282 Activity app not loading properly
* [x] #10385 Console error when page has no search
* [x] #10391 Fix app navigation controller to return an array
* [x] #10381  Avatar in author row of comments gets resized to 16x16 px
* [ ] #9242 Fix display name ignored when creating new user
* [ ] #10361 Public share note needs a better ui
* [ ] #10398 Opening details after coming from favorites view is broken
* ~~#10212 Calendar ressource and room manager should not take implementations but class names (technical debt/public API)~~ (does not block beta 1, but should be looked at during the bug fixing phase)
* ~~#5456 Privacy: Avatar public available~~ (does not block beta 1, but should be looked at during the bug fixing phase)

### CODE REVIEW

**Comment by MorrisJobke** [comment]
> #10248 Introduces avatars for guest users

@skjnldsv @nickvergessen I added it to the "optional" section, but it might need to go to the must haves, right?

**Comment by nickvergessen** [comment]
We didn't have "normal" guest avatars for so long, doesn't matter to not have them one more year.

**Comment by skjnldsv** [comment]
> one more year.

Year?? :scream:

**Comment by nickvergessen** [comment]
Version, sorry....

**Comment by juliushaertl** [comment]
Added: 

> https://github.com/nextcloud/activity/pull/282 Activity app not loading properly

**Comment by MorrisJobke** [comment]
> #10398 Opening details after coming from favorites view is broken

That is the failing acceptance test.

**Review by rullzer — APPROVED** [review]

**Review by nickvergessen — APPROVED** [review]

File: version.php [hunk 1]
```diff
@@ -29,10 +29,10 @@
 // between betas, final and RCs. This is _not_ the public version number. Reset minor/patchlevel
 // when updating major/minor version number.
 
-$OC_Version = array(14, 0, 0, 12);
+$OC_Version = array(14, 0, 0, 13);
 
 // The human readable string
-$OC_VersionString = '14.0.0 alpha';
+$OC_VersionString = '14.0.0 Beta 1';
```
[the remark this answers is not in the document]
Comment (MorrisJobke) [reply version.php]: Was like that in the past:

https://github.com/nextcloud/server/commit/88d5f6ec637fd7b3e540b0616e0283b747a79969#diff-5bbd3ccd25214f9956eea7e9f714bc08R35

https://github.com/nextcloud/server/commit/62656dc5c2b39f911fa3260ac291f0301b03a7a3#diff-5bbd3ccd25214f9956eea7e9f714bc08R35

**Pull request merged**